### PR TITLE
WIP AI upgrade in function generate_content and stock endpoint and keyap…

### PR DESCRIPTION
**Pull request by lamrani002**
# NEW|New updating in script for handle call ia api

### CODE REVIEW

**Comment by eldy** [comment]
@lamrani002 Can you pull and repush the PR ?

**Comment by lamrani002** [comment]
Ok I will do it tomorrow

File: htdocs/ai/ajax/generate_content.php [hunk 1]
```diff
@@ -41,9 +41,9 @@
 	define('NOREQUIRESOC', '1');
 }
 
-require_once '../../main.inc.php';
-require_once DOL_DOCUMENT_ROOT.'/ai/class/ai.class.php';
+require '../../main.inc.php';
 
+require_once DOL_DOCUMENT_ROOT.'/ai/class/ai.class.php';
```
[the remark this answers is not in the document]
Comment (lamrani002) [reply htdocs/ai/ajax/generate_content.php:46]: when i add top_httphead('application/json'); i get a error parsing but i added in ajax function => contentType: 'application/json', and it works properly

**Comment by eldy** [comment]
Something looks wrong with PR: 411 files were modified. Did you run an update of your fork before pushing the PR ?

**Pull request closed without merging**